src/main/java/lotto/domain/Winnings.java

```diff
@@ -0,0 +1,35 @@
+package lotto.domain;
+
+public enum Winnings {
+
+    사등("3개", 5000),
+    삼등("4개", 50000),
+    이등("5개", 1500000),
+    일등("6개", 2000000000);
```

Comment:
```suggestion
    일등("6개", 2_000_000_000);
```

자바7 부터 숫자에 언더스코어(_) 를 사용할 수 있습니다.
천의 자리를 끊어서 좀 더 가독성 있게 사용해 보는것은 어떨까요 ?! 😄